src/GroupManager.php

```diff
@@ -459,6 +470,12 @@ protected function refreshGroupRelationMap() {
 
     foreach ($this->entityTypeBundleInfo->getAllBundleInfo() as $group_content_entity_type_id => $bundles) {
       foreach ($bundles as $group_content_bundle_id => $bundle_info) {
+
+        if ($group_content_bundle_id == 'user') {
```

Comment:
we've seen we might have multiple user bundles, so probably this needs a better check.